release.py

```diff
@@ -204,15 +204,13 @@ def main():
     old_version = get_latest_remote_version()
     is_old_version_published = is_published_release(old_version)
 
-    skip_version_bump = args.skip_version_bump
     if not is_old_version_published and not args.force:
-        print(f"Skipping version bump.\nThe latest tag {old_version} does not correspond to a published github release." +
+        die(f"The latest tag {old_version} does not correspond to a published github release." +
         " It may be a draft release or it may have never been created." +
         " If you still want to upgrade the version, rerun the script with --force.")
-        skip_version_bump = True
 
     new_version = old_version
-    if not skip_version_bump:
+    if not args.skip_version_bump:
```

Comment:
wdyt of getting rid of `--skip_version_bump` and replacing it with `--bump_version_type=none`? Fine to do it in a follow-up PR if you want.